manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name" : "Multiqc",
+    "description" : "Aggregate bioinformatics results across many samples into a single report.",
+    "version": "1.0",
+    "type": "job",
+    "contacts" : [],
+    "regovar_db_access": False,
```

Comment:
Booleans in JSON must be lowercase. See [RFC7159](https://tools.ietf.org/html/rfc7159), 3. Values:

    A JSON value MUST be an object, array, number, or string, or one of
    the following three literal names:

       false null true

    The literal names MUST be lowercase.  No other literal names are
    allowed.

       value = false / null / true / object / array / number / string
       false = %x66.61.6c.73.65   ; false
       null  = %x6e.75.6c.6c      ; null
       true  = %x74.72.75.65      ; true

If the documentation uses capitalized booleans like in Python, it must be fixed.

Reply:
yes, this error come from the example file of the tutorial. but now the doc must refere this repository instead of tutorials additional assets files.
I will update the doc and tutorials when this repo will be ready to use